community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/RecordFormats.java

```diff
@@ -58,5 +68,22 @@ public Factory( String key, String... altKeys )
 
     RecordFormat<RelationshipTypeTokenRecord> relationshipTypeToken();
 
-    RecordFormat<RelationshipGroupRecord> relationshipGroup();
+    RecordFormat<DynamicRecord> dynamic();
+
+    boolean hasStore( StoreType store );
```

Comment:
Oh actually, that thing isn't even needed anymore since the direct store migrator will not used between formats that aren't logically compatible, which implies that they have the same set of stores as well.